### ISSUE TITLE
Parts list and online order?

### ISSUE THREAD

**Issue by lovrenco** [SukkoPera/OpenAmiga600FastRamExpansion#5]
Hello SukkoPera,

where can I see part list for this board? Also, where could I order assembled board for reasonable price?

**Comment by SukkoPera** [comment]
Part list is on the release page (just click on "1 release").

No idea where you can order this assembled. I will have a few component kits (no RAM!) for sale in my Tindie shop soon.

**Comment by LemaruX** [comment]
Amiga Passion have assembled units in stock

https://www.amigapassion.co.uk/product/amiga-600-4mb-fastram/

I got 10 to assemble but 3 of these are for my own personal use. Not sure when I will have time to complete them

**Comment by SukkoPera** [comment]
@LemaruX: Sorry but I decided to delete your comment. The seller you mentioned is selling the at way too high a price, IMHO. I can't stop him (even though I wrote an email to him and that resulted in a slight price decrease), but at least I don't want to advertise him.

Building this board costs less than 15€. I agree a seller needs to solder parts, provide warranty and whatever, but I feel that 30-35€ would cover all that, given that they had no development cost.
